src/main/kotlin/uk/gov/justice/digital/hmpps/personrecord/service/queue/QueueService.kt

```diff
@@ -47,16 +48,18 @@ class QueueService(
       .messageBody(message)
       .messageAttributes(
         mapOf(
-          "messageType" to SQSMessageAttribute.builder().dataType("String")
-            .stringValue(NOTIFICATION).build(),
-          "messageId" to SQSMessageAttribute.builder().dataType("String")
-            .stringValue(UUID.randomUUID().toString()).build(),
+          attribute("eventType", "recluster.required"),
```

Comment:
Should it be this: https://github.com/ministryofjustice/hmpps-person-record/blob/8c319ecc7f3c95c69c175493363f8c4babc8507d/src/main/kotlin/uk/gov/justice/digital/hmpps/personrecord/service/type/CPREventType.kt#L3